docs/src/modules/components/MarkdownElement.js

```diff
@@ -31,17 +33,14 @@ const Root = styled('div')(({ theme }) => ({
     display: 'inline-block',
     fontFamily: 'Consolas, "Liberation Mono", Menlo, Courier, monospace',
     WebkitFontSmoothing: 'subpixel-antialiased',
-    padding: '0 3px',
+    padding: '0 5px',
     color: theme.palette.text.primary,
-    backgroundColor:
-      theme.palette.mode === 'light'
-        ? 'rgba(255, 229, 100, 0.2)'
-        : alpha(theme.palette.primary.main, 0.08),
-    fontSize: '.85em',
-    borderRadius: 2,
+    backgroundColor: alpha(theme.palette.primary.light, 0.15),
+    fontSize: '.8125em',
+    borderRadius: theme.shape.borderRadius,
   },
   '& code[class*="language-"]': {
-    backgroundColor: '#272c34',
+    backgroundColor: "#001E3C",
```

Comment:
We should check why sometimes using `theme.palette.primaryDark[700]` is throwing. cc @siriwatknp

Reply:
I might have missed it.